mech_client/acn.py

```diff
@@ -66,27 +67,41 @@
     }
 ]
 
-IPFS_REQUEST_URL = "https://gateway.autonolas.tech/ipfs/{hash_str}/{request_id}"
```

Comment:
Where has this gone?

Reply:
This was unused, the URL is directly being formatted here https://github.com/valory-xyz/mech-client/blob/main/mech_client/acn.py#L138